airbyte-integrations/connectors/destination-mysql/src/test-integration/java/io/airbyte/integrations/destination/mysql/MySQLDestinationAcceptanceTest.java

```diff
@@ -43,6 +43,7 @@
 import org.junit.jupiter.api.Timeout;
 import org.testcontainers.containers.MySQLContainer;
 
+@Disabled
```

Comment:
everything starts failing because we're writing to a different raw table. Disable the legacy DAT suite.

... https://github.com/airbytehq/airbyte/issues/29818 continues to exist